src/test/scala/tech/cryptonomic/conseil/metadata/TestPlatformDiscoveryOperations.scala

```diff
@@ -0,0 +1,35 @@
+package tech.cryptonomic.conseil.metadata
+
+import tech.cryptonomic.conseil.generic.chain.{DataTypes, PlatformDiscoveryOperations, PlatformDiscoveryTypes}
+import tech.cryptonomic.conseil.generic.chain.PlatformDiscoveryTypes.{Attribute, Entity}
+
+import scala.concurrent.Future
+
+class TestPlatformDiscoveryOperations extends PlatformDiscoveryOperations {
+
+  var entities: List[Entity] = List.empty
+  var attributes: List[Attribute] = List.empty
+
+  def addEntity(entity: Entity): Unit = entities = entities :+ entity
```

Comment:
maybe add `this.syncronized{}` to those operations?

Reply:
done